doc/liblouis.texi

```diff
@@ -835,6 +851,99 @@ numericmodechars .,
 midendnumericmodechars -/
 @end example
 
+Note: This opcode was implemented in Liblouis specifically in order
+to support Unified English Braille (UEB). It may be used in any table,
+but may have unpredicted side-effects if used outside the intended
+context. Use with great care, and test thoroughly.
+
+@end table
+
+@node Opcodes for Standing Alone Sequences
+@section Opcodes for Standing Alone Sequences
+
+The term "standing alone" comes from the specification of Unified
```

Comment:
Maybe we should emphasize the term standing alone see the [texinfo manual](https://www.gnu.org/software/texinfo/manual/texinfo/html_node/_0040emph-_0040strong.html)), e.g. like this
``` texinfo
@emph{standing alone}
```
Actually come to think of it, maybe we should even add it to the concept index. Add the following before the paragraph that you want to link in the index:
``` texinfo
@cindex Standing alone
```